hack/deploy-karmada-agent.sh

```diff
@@ -4,24 +4,66 @@ set -o errexit
 set -o nounset
 
 REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
-KARMADA_APISERVER_KUBECONFIG=${KARMADA_APISERVER_KUBECONFIG:-"/var/run/karmada/karmada-apiserver.config"}
 
-# The host cluster name which used to install karmada control plane components.
-MEMBER_CLUSTER_NAME=${MEMBER_CLUSTER_NAME:-"member3"}
-MEMBER_CLUSTER_KUBECONFIG=${MEMBER_CLUSTER_KUBECONFIG:-"${HOME}/.kube/member3.config"}
-
-source ${REPO_ROOT}/hack/util.sh
 function usage() {
   echo "This script will deploy karmada agent to a cluster."
-  echo "Usage: hack/deploy-karmada-agent.sh"
-  echo "Example: hack/deploy-karmada.sh"
+  echo "Usage: hack/deploy-karmada-agent.sh <KARMADA_APISERVER_KUBECONFIG> <KARMADA_APISERVER_CONTEXT_NAME> <MEMBER_CLUSTER_KUBECONFIG> <MEMBER_CLUSTER_CONTEXT_NAME>"
+  echo "Example: hack/deploy-karmada-agent.sh ~/.kube/karmada.config karmada-apiserver ~/.kube/karmada.config member1"
```

Comment:
Just a question, who are the primary users of the `deploy-karmada-agent.sh`?  Karmada CI or Karmada users?

Reply:
Oh, I see. it will be used by `karmada-bootstrap.sh`.

Reply:
So far I think that `deploy-karmada-agent.sh` will be used by `karmada-bootstrap.sh`, but I feel that it can be provided for Karmada users to install their karmada agent. 
For involving two clusters (the one is karmada api-server, another is member cluster), I have no idea about optimizing the argument. Need an idea here